### ISSUE TITLE
Test update of Kafka to latest version

### ISSUE THREAD

**Issue by tnavatar** [clingen-data-model/expediter#1]
We're currently on version 0.10 of Kafka, version 2.2 is available. Kafka maintains backward and forward compatibility with clients, but we haven't tested the best approach to maintain client state between clusters. This includes the content of topics, as well as consumer offsets. The rest of the state, including topics and permissions, is stored in this repository.

There is a tool available, Kafka Mirrormaker, which may be useful in spinning up a new cluster based on an existing one.

**Comment by larrybabb** [comment]
Is this something @toneillbroad should help with? I think it would be good to delegate some items (like this) to others so you can stay focused on the Actionability and Gene Validity design and development which is less delagate-able.

**Comment by tnavatar** [comment]
Yup! Or Phil as well. I'm making a record of these tasks so that others can take them on.
